### ISSUE TITLE
Textfield long press paste will be blocked by the status bar.

### ISSUE THREAD

**Issue by CaiJingLong** [flutter/flutter#24088]
Copy and paste pop-up boxes will only appear above the textfield cursor, and if the textfield is below the status bar, it will be blocked by the status bar and can not be touched. 
If the screen shape is like the iPhone X, the naked eye will not see it.

like the screenshot 
![](https://ws1.sinaimg.cn/large/844036b9ly1fx0hevevm7j20u0078my5.jpg)

------

flutter doctor -v 

```
[✓] Flutter (Channel dev, v0.10.2, on Mac OS X 10.13.6 17G65, locale zh-Hans-CN)
    • Flutter version 0.10.2 at /Users/cai/fluttersdk/flutter
    • Framework revision d8cbb80206 (13 days ago), 2018-10-26 01:30:21 -0400
    • Engine revision 6c2ade9fa2
    • Dart version 2.1.0-dev.8.0.flutter-bf26f760b1

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.2)
    • Android SDK at /Users/cai/Library/Android/sdk
    • Android NDK at /Users/cai/Library/Android/sdk/ndk-bundle
    • Platform android-28, build-tools 28.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 9.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.2, Build version 9C40b
    • ios-deploy 1.9.2
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 29.0.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] IntelliJ IDEA Ultimate Edition (version 2018.1.4)
    • IntelliJ at /Users/cai/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 27.1.2
    • Dart plugin version 181.4892.1

[✓] VS Code (version 1.28.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.20.0

[✓] Connected device (1 available)
    • MI 8 • b6ec3684 • android-arm64 • Android 8.1.0 (API 27)

```

**Comment by zoechi** [comment]
Duplicate of #25066 ?
Related to #3945
